src/style/listOfItems.css

```diff
@@ -1,6 +1,78 @@
 .listItem{
     width: 225px;
-    height: 120px;
-    background-color: white;
+    height: 110px;
+    background-color: var(--main-color);
     margin-right: 5px;
+    overflow:hidden;
+    cursor: pointer;
 }
+.listItem .movieCover{
+    width: 100%;
+    height: 100%;
+    border-radius: 5px;
+    object-fit: cover;
+}
+.itemInfo video{
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    position: absolute;
+    top: 0;
+    left: 0;
+    border-radius: 5px;
+    z-index: 0;
+}
+/* .listItem img{
+    height: 140px;
+} */
+.itemInfo .movieInfo{
+    position: absolute;
+    z-index: 1;
+    background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0), rgb(0, 0, 0));
+    width: 100%;
+    height:  100%;
+}
+.itemInfo:hover{
+    width: 300px;
+    height: 350px;
+    position: absolute;
+    top: -200px;
+    -webkit-box-shadow: 0px 0px 15px 0px rgba(255,255,255,0.07); 
+    box-shadow: 0px 0px 15px 0px rgba(255,255,255,0.07);
+    background-color: rgb(22, 19, 19);
+    border-radius: 5px;
+    z-index: 1;
+}
+.itemInfoTop{
+    display: flex;
+    align-items: left;
+    justify-content:space-between;
+    padding-top:95%;
+    padding-left: 20px;
+    padding-right: 40px;
+    font-size: 15px;
+}
+.itemInfoTop .match{
+    color: yellowgreen;
+}
+.movieInfo .genre{
+    align-items: center;
+    display: flex;
+    justify-content: left;
+    padding-left: 20PX;
+    font-size: 15px;
+}
+.movieInfo .icons{
+    display: flex;
+    align-items: left;
+    justify-content: space-between;
+    padding-left: 20px;
+    padding-top: 80%;
+    position: absolute;
```

Comment:
why absolute?

Reply:
It will not affect the responsiveness, it is use for hover in order to stay on the top of the object or Movie image he want to hover